### ISSUE TITLE
Linker warnings shown while generating CPG for `x42` sample project

### ISSUE THREAD

**Issue by ursachec** [joernio/joern#549]
```
2021-06-22 12:24:33.480 WARN Linker: Could not create edge. Destination lookup failed. edgeType=EVAL_TYPE, srcNodeType=CALL, srcNodeId=1000106, dstNodeType=TYPE, dstFullName=
2021-06-22 12:24:33.481 WARN Linker: Could not create edge. Destination lookup failed. edgeType=EVAL_TYPE, srcNodeType=CALL, srcNodeId=1000107, dstNodeType=TYPE, dstFullName=
2021-06-22 12:24:33.482 WARN Linker: Could not create edge. Destination lookup failed. edgeType=EVAL_TYPE, srcNodeType=CALL, srcNodeId=1000110, dstNodeType=TYPE, dstFullName=
2021-06-22 12:24:33.482 WARN Linker: Could not create edge. Destination lookup failed. edgeType=EVAL_TYPE, srcNodeType=CALL, srcNodeId=1000111, dstNodeType=TYPE, dstFullName=
2021-06-22 12:24:33.482 WARN Linker: Could not create edge. Destination lookup failed. edgeType=EVAL_TYPE, srcNodeType=CALL, srcNodeId=1000112, dstNodeType=TYPE, dstFullName=
2021-06-22 12:24:33.482 WARN Linker: Could not create edge. Destination lookup failed. edgeType=EVAL_TYPE, srcNodeType=CALL, srcNodeId=1000118, dstNodeType=TYPE, dstFullName=
2021-06-22 12:24:33.482 WARN Linker: Could not create edge. Destination lookup failed. edgeType=EVAL_TYPE, srcNodeType=CALL, srcNodeId=1000121, dstNodeType=TYPE, dstFullName=
2021-06-22 12:24:33.482 WARN Linker: Could not create edge. Destination lookup failed. edgeType=EVAL_TYPE, srcNodeType=CALL, srcNodeId=1000123, dstNodeType=TYPE, dstFullName=
2021-06-22 12:24:33.482 WARN Linker: Could not create edge. Destination lookup failed. edgeType=EVAL_TYPE, srcNodeType=CALL, srcNodeId=1000125, dstNodeType=TYPE, dstFullName=
```

**Comment by weizhigp** [comment]
New to the joern，I also encountered this problem and wondered if I could ignore it

**Comment by kevinGit777** [comment]
Same here

**Comment by ursachec** [comment]
@weizhigp @kevinGit777 sorry for the late reply, yes, the warnings are safe to ignore

**Comment by ursachec** [comment]
Fixed in Joern `v1.1.194`
